ci/checks/copyright.py

```diff
@@ -38,6 +38,9 @@
     re.compile(r"[.]flake8[.]cython$"),
     re.compile(r"meta[.]yaml$")
 ]
+ExemptFiles = [
+    re.compile(r"_thirdparty")
+]
```

Comment:
Can we make this a command line option? Would be nice to have this feature controllable from the command line. Something like `--except` should work. And by default it would be `--except python/cuml/_thirdparty` but you could add others.